src/app/views/sidebar/sidebar.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../../styles/variables';
```

Comment:
Implement these styles using css-in-js

Reply:
Handled in [adds sample queries to the view](https://github.com/microsoftgraph/microsoft-graph-explorer-v2/pull/136)